README.md

```diff
@@ -141,6 +141,21 @@ allowedTags: [],
 allowedAttributes: []
 ```
 
+### "What if I want to allow only specific values on some attributes?"
+
+When configuring the attribute in `allowedAttributes` simply use an object with attribute `name` and an allowed `values` array, this will allow any combination of the `values` in the array where the `values` are separated by a space. In the following example `sandbox="allow-forms allow-modals allow-orientation-lock allow-pointer-lock allow-popups allow-popups-to-escape-sandbox allow-scripts"` would become `sandbox="allow-popups allow-scripts"`:
+
+```js
+        allowedAttributes: {
+          iframe: [ 
+            {
+              name: 'sandbox',
+              values: ['allow-popups', 'allow-same-origin', 'allow-scripts']
+            }
+          ]
+```
+Note: If you only want to allow a single specific attribute value, then `values` should only have a single string in the array.
```

Comment:
This does not solve the problem. What if the allowed values themselves contain spaces? The splitting behavior must be optional.

Reply:
Understood, added `multiple` option and tests for matching against a list of values and for exact matching.